### PR TITLE
🦋 Update Buyer accounts' table header

### PR DESCRIPTION
**Pull request by josemigallas**
[THREESCALE-9764: PF4 to buyer accounts table header](https://issues.redhat.com/browse/THREESCALE-9764)

Before:
Screenshot 2023-06-27 at 12 22 14

After:
Screenshot 2023-06-26 at 17 47 22
Screenshot 2023-06-26 at 17 47 41
Screenshot 2023-06-26 at 17 47 50

### CODE REVIEW

File: app/javascript/packs/provider.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // consistent styles throughout the application, PatternFly Base CSS should be used in the project,
 // or some components may diverge in appearance. This import statement should be placed before your
 // main application component is imported.
-@import '~@patternfly/react-core/dist/styles/base.css';
+@import '~@patternfly/patternfly/patternfly-base.css';
```
Comment (josemigallas) [inline app/javascript/packs/provider.scss:5]: This base package also includes the icons, which `patternfly/react-core` gets from `patternfly/react-icons`.

File: app/helpers/column_sorting_helper.rb [hunk 2]
```diff
@@ -30,6 +31,43 @@ def sortable(column, title = nil, path = :url_for)
             { :class => css_class }
   end
 
+  def th_sortable(column, title = nil, path = :url_for)
```
Comment (josemigallas) [inline app/helpers/column_sorting_helper.rb:34]: This will eventually replace the old method `sortable` once all tables have been implemented using PF4.

Comment (mayorova) [reply app/helpers/column_sorting_helper.rb:34]: I think this might need a unit test?

**Comment by mayorova** [comment]
The screenshots don't show that, but actually the filters are misplaced:

Screenshot from 2023-06-27 13-20-38

Seems like it's missing an empty `<td>` in the Signup Date column.

Also, I think it would be nice if Create and Activate buttons were aligned.

**Review by mayorova — CHANGES_REQUESTED** [review]
My change requests here: https://github.com/3scale/porta/pull/3441#issuecomment-1609307883

File: app/helpers/column_sorting_helper.rb (hunk 2 above)
Comment (josemigallas) [reply app/helpers/column_sorting_helper.rb:34]: In all seriousness, I don't think it's worth it. The method (and the original) are to fiddly and IMHO it should be enough to test it indirectly (via 🥒)

File: app/views/buyers/accounts/_search_results.html.slim [hunk 3]
```diff
@@ -71,7 +68,7 @@ div class="pf-c-card"
             = account_states_info(account)
           td role="cell" class="pf-c-table__action actions operations"
             ul.Actions
-              - if account.admin_user.try!(:pending?)
+              - if account.admin_user.try!(:pending?) || true
```
Comment (mayorova) [inline app/views/buyers/accounts/_search_results.html.slim:71]: :eyes:

Comment (josemigallas) [reply app/views/buyers/accounts/_search_results.html.slim:71]: 😭

**Review by mayorova — APPROVED** [review]

**Pull request merged**